src/mlagility/cli/report.py

```diff
@@ -1,521 +1,204 @@
-    printing.log("Summary spreadsheet saved at ")
-    printing.logn(str(spreadsheet_file), printing.Colors.OKGREEN)
+    # Description: License of the model
+    # Source: Specified as part of the .py file using the label "license"
+    license: str = "-"
 
-    # Make the logs.yaml file
-    with open(results_file, "w", encoding="utf8") as outfile:
-        yaml.dump(results_dict, outfile)
+    # Description: Task being performed by the model
+    # Source: Specified as part of the .py file using the label "task"
+    task: str = "-"
 
-    # Print message with the output file path
-    printing.log("Build results across all models collected at ")
-    printing.logn(str(results_file), printing.Colors.OKGREEN)
+    # Description: Number of chips needed to run the model when mapped to Groq
+    # Source: Computed by GroqFlow
+    groq_chips_used: str = "-"
 
+    # Description: Number of chips needed to run the model when mapped to Groq
+    # Source: Computed by GroqFlow
+    groq_chips_used: str = "-"
 
-def op_spreadsheet(cache_dir):
+    # Description: An estimation of latency, in ms, to run one invocation of the
+    #              model on a GroqNode server (includes I/O).
+    # Source: computed by GroqFlow
+    groq_estimated_latency: str = "-"
 
-    # Declare file paths
-    spreadsheet_file = os.path.join(cache_dir, op_support_filename)
+    # Description: Mean total latency, in ms, to run one invocation using TensorRT.
+    # Source: benchit.benchmark() on 100 runs
+    nvidia_latency: str = "-"
 
-    with open(spreadsheet_file, "w", newline="", encoding="utf8") as spreadsheet:
-        writer = csv.writer(spreadsheet)
+    # Description: Mean total latency, in ms, to run one invocation using OnnxRuntime.
+    # Source: benchit.benchmark() on 100 runs
+    x86_latency: str = "-"
+
+
+def summary_spreadsheet(args) -> None:
+
+    # Input arguments from CLI
+    cache_dirs = [os.path.expanduser(dir) for dir in args.cache_dirs]
+    report_dir = os.path.expanduser(args.report_dir)
 
-        # Git commit placeholder
-        writer.writerow(
-            [
-                "Git Commit: ",
-            ]
-        )
+    # Name report file
+    report_path = os.path.join(report_dir, get_report_name())
 
-        writer.writerow(
-            [
-                "Model Name",
-                "Unsupported ops (Optimized ONNX)",
-            ]
-        )
+    # Create report dict
+    Path(report_dir).mkdir(parents=True, exist_ok=True)
+    report = {}
 
+    # Add results from all cache folders
+    for cache_dir in cache_dirs:
+
+        # List all yaml files available
         all_model_state_yamls = cache.get_all(path=cache_dir, file_type="state.yaml")
         all_model_state_yamls = sorted(all_model_state_yamls)
 
+        # Add each model to report
         for model_state_yaml in all_model_state_yamls:
+
+            # Models are identified my the build name
+            build_name = model_state_yaml.split("/")[-2]
```

Comment:
Move this down a line and use `state.config.build_name` instead

Reply:
Changed!